web/views.py

```
@@ -7018,3 +7021,97 @@ def users_list(request: HttpRequest) -> HttpResponse:
+        return JsonResponse({"error": "Only POST method allowed"}, status=405)
+
+    try:
+        topic = ForumTopic.objects.get(pk=pk)
+        vote_type = request.POST.get("vote_type")
+
+        if vote_type not in ["up", "down"]:
+            return JsonResponse({"error": "Invalid vote type"}, status=400)
+
+        # Check if user already voted on this topic
+        vote, created = ForumVote.objects.get_or_create(
+            user=request.user, topic=topic, defaults={"vote_type": vote_type}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)